src/test/resources/application.conf

```diff
@@ -46,17 +46,17 @@ interestForecastingApi {
   alertsToIgnore: []
 }
 
-StatementOfLiabilityApi {
+statementOfLiabilityApi {
   ignoreOptimizelyAlerts: true
   failureThreshold: "Medium"
-  testUrl: "http://localhost:0000/statement-of-liability"
-  contextBaseUrlRegex: "http://localhost:0000.*"
-  alertUrlsToReport: ["http://localhost:0000"]
+  testUrl: "http://localhost:9945/statement-of-liability/hello-world"
+  contextBaseUrlRegex: "http://localhost:9945.*"
+  alertUrlsToReport: ["http://localhost:9945"]
   testingAnApi = false
```

Comment:
IFS Zap config has "testingAnApi" set to true and SoL Zap config has "testingAnApi" set to false?

Reply:
the test works with true or false but have changed it to true incase it affects later testing